### ISSUE TITLE
Migrate to null safety

### ISSUE THREAD

**Issue by kozw** [firstfloorsoftware/giphy_picker#18]
Awaiting null safety in http dependency: https://github.com/dart-lang/http/issues/501

**Comment by sameeerkashyap** [comment]
@kozw  I would like to work on this if it's up for grabs! once the dependencies are resolved.

**Comment by kozw** [comment]
Appreciate that, but work on null safety is mostly complete. See the null-safety branch.

**Comment by sameeerkashyap** [comment]
@kozw  great !, thanks for letting me know.

**Comment by amantoux** [comment]
```
# temporarily using git dependency, http null safety is not ready yet
    git: https://github.com/dart-lang/http
```
FYI
http: 0.13.0 is null safety compliant

**Comment by kozw** [comment]
Thanks! Just published 2.0.0-nullsafety.0
